libhif/hif-repo.c

```diff
@@ -882,6 +885,34 @@ hif_repo_setup(HifRepo *repo, GError **error)
         return FALSE;
     if (!lr_handle_setopt(priv->repo_handle, error, LRO_GNUPGHOMEDIR, priv->keyring))
         return FALSE;
+
+    if (g_key_file_has_key(priv->keyfile, priv->id, "sslverify", NULL)) {
+        if (g_key_file_get_boolean(priv->keyfile, priv->id, "sslverify", NULL)) {
+            if (!lr_handle_setopt(priv->repo_handle, error, LRO_SSLVERIFYPEER, TRUE))
+                return FALSE;
+            if (!lr_handle_setopt(priv->repo_handle, error, LRO_SSLVERIFYHOST, TRUE))
+                return FALSE;
+        }
+    }
+
+    sslcacert = g_key_file_get_string(priv->keyfile, priv->id, "sslcacert", NULL);
```

Comment:
This is so boring and mechanical that I whether really the repo file parsing belongs in...well, lib*repo*.  But we can revisit that later :clock1: